src/components/ebay-pagination/examples/2-fake-link/template.marko

```diff
@@ -0,0 +1,13 @@
+<ebay-pagination hijax>
```

Comment:
Rename to `2-hijax`

Reply:
Rename the directory from `2-fake-link` to `2-hijax`, right?